core/datastore/src/main/kotlin/org/meshtastic/core/datastore/UiPreferencesDataSource.kt

```diff
@@ -57,6 +58,8 @@ class UiPreferencesDataSource @Inject constructor(private val dataStore: DataSto
 
     val nodeSort: StateFlow<Int> = dataStore.prefStateFlow(key = NODE_SORT, default = -1)
     val includeUnknown: StateFlow<Boolean> = dataStore.prefStateFlow(key = INCLUDE_UNKNOWN, default = false)
+    val includeInfrastructure: StateFlow<Boolean> =
+        dataStore.prefStateFlow(key = INCLUDE_INFRASTRUCTURE, default = true)
```

Comment:
```suggestion
        dataStore.prefStateFlow(key = EXCLUDE_INFRASTRUCTURE, default = false)
```
Let's make this opt-in - I don't want to field the questions